EasyInventory/src/test/java/com/easyinventory/enterprise/EnterpriseApplicationTests.java

```diff
@@ -13,7 +13,7 @@
 class EnterpriseApplicationTests {
 
     @Autowired
-    EasyInventoryService easyInventoryService;
+    private EasyInventoryService easyInventoryService;
```

Comment:
Based upon java standards, I opted to add a private access modifier to this variables, as they are only accessed in this class.

https://docs.oracle.com/javase/tutorial/java/javaOO/variables.html